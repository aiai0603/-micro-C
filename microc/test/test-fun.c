Int fact(Int i){
    if(i == 1){
        return 1;
    }else{
        return i * fact(i - 1);
    }
}
Int main(){
    Int n;
    n=4;
    print("%d", fact(n));
}