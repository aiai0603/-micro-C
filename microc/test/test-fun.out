24 19 0 29 25 13 11 0 1 6 17 16 0 1 21 1 13 11 13 11 0 1 2 19 1 5 3 21 1 15 1 13 0 4 12 15 -1 13 11 19 1 5 22